aws/rapids_ec2_mnmg.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>... We verify the columns we need have appropriate datatypes to make it ready for computation using cuML.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>